index.html

```diff
@@ -3,31 +3,32 @@
   <head>
     <meta charset="utf-8">
     <meta name="description" content="Hello Studio Website.">
+    <link rel="stylesheet" href="style/main.css">
     <title>Hello Studio</title>
   </head>
   <body>
-    <header>
-      <h1>
-        <span>the time is</span>
-        <span>now</span>
+    <header class="header">
+      <h1 class="header__heading">
+        <span class="header__heading1">the time is</span>
+        <span class="header__heading2">now</span>
       </h1>
-      <nav>
-        <ul>
-          <li><a href="#intro">intro</a></li>
-          <li><a href="#gallery">gallery</a></li>
-          <li><a href="#services">services</a></li>
-          <li><a href="#contact">contact</a></li>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#intro" class="nav__link">intro</a></li>
```

Comment:
Ewentualnie `nav__list-item`, wtedy wiadomo, że to będzie element listy. Ale to tylko moja preferencja, więc możesz zostawić tak jak wolisz 👍

Reply:
Po porównaniu dochodzę do wniosku, że Twoja preferencja bardziej mi odpowiada :)